mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -842,6 +842,9 @@ func expandRegionConfigAutoScaling(tfList []interface{}) *matlas.AdvancedAutoSca
 		apiObject3.MaxInstanceSize = v.(string)
 	}
 
+	apiObject.DiskGB = apiObject2
+	apiObject.Compute = apiObject3
```

Comment:
This name is not helpful. Could we rename `apiObject` to `advancedAutoScaling`, `apiObject2` to `diskGB` and `apiObject3` to `compute` ?